cypress/commons/actions/generic/Scenarios.js

```diff
@@ -39,6 +39,17 @@ function getDashboardPlaceholder() {
   return cy.get(GENERIC_SELECTORS.scenario.dashboard.placeholder);
 }
 
+// Open scenario creation dialog
+function openScenarioCreationDialog() {
+  getScenarioCreationButton().click();
+}
+
+// From scenario creation dialog, select a run template
+function selectRunTemplate(runTemplate) {
+  cy.focused().clear().type(runTemplate);
```

Comment:
I'm not a huge fan of `cy.focused` in this function that should be "generic" and callable no matter the context.
Here it works, cause you use this function in `CreateScenario-tests.spec.js` just before putting the focus on runTemplate selectBox.
Can you use a selector instead of `cy.focused` please?

Reply:
Nice catch
+1 to use a selector